product_docs/docs/pgd/5/cli/command_ref/pgd_set-node-options.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: set-node-options
+---
+
+Sets node options such as route_fence, route_priority, and route_writes.
```

Comment:
Fixed but instead of back ticks used single quote.

Reply:
BTW unlike long description, in short we could use back ticks but for consistency keeping it single quote every where because same is also viewed in terminal help.

```
postgres@bdr-a1:/home/admin $ pgd

The EDB Postgres Distributed Command Line Interface (PGD CLI) is a tool to
manage your EDB Postgres Distributed cluster. It allows you to run commands
against EDB Postgres Distributed clusters. You can use it to inspect and manage
cluster resources.

Usage:
  pgd [command]

Available Commands:
  check-health       Checks the health of the EDB Postgres Distributed cluster.
  completion         Generate the autocompletion script for the specified shell
  create-proxy       Creates proxy in the EDB Postgres Distributed cluster.
  delete-proxy       Deletes proxy from the EDB Postgres Distributed cluster.
  help               Help about any command
  set-group-options  Sets group options such as 'enable_raft', 'enable_proxy_routing', and 'location'.
  set-node-options   Sets node options such as 'route_fence', 'route_priority', and 'route_writes'.
  set-proxy-options  Sets proxy options such as 'listen_address', 'listen_port', and 'max_client_conn'.
  show-clockskew     Shows the status of clock skew between each BDR node pair.
  show-events        Shows events such as background worker errors and node membership changes.
  show-groups        Shows all groups in the EDB Postgres Distributed cluster and their summary.
  show-nodes         Shows all nodes in the EDB Postgres Distributed cluster and their summary.
  show-proxies       Shows all proxies in the EDB Postgres Distributed cluster and their summary.
  show-raft          Shows BDR Raft (consensus protocol) details.
  show-replslots     Shows the status of BDR replication slots.
  show-subscriptions Shows BDR subscription (incoming replication) details.
  show-version       Shows the version of BDR and Postgres installed on each node.
  switchover         Switches over to new write leader.
  verify-cluster     Verifies whether the cluster follows the rules as per the AlwaysOn architecture.
  verify-settings    Verifies the EDB Postgres Distributed cluster settings.

Flags:
  -f, --config-file string   config file; ignored if 
                             --dsn flag is present (default "/etc/edb/pgd-cli/pgd-cli-config.yml")
      --dsn string           database connection string
                             e.g."host=bdr-a1 port=5432 dbname=bdrdb user=postgres "
  -h, --help                 help for pgd
  -L, --log-level string     logging level: debug, info, warn, error (default "error")
  -o, --output string        output format: json, yaml
  -v, --version              version for pgd

Use "pgd [command] --help" for more information about a command.

```